api/src/certification/enrolment/domain/models/Subscription.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @typedef {import ('../../../shared/domain/models/SubscriptionTypes.js').SubscriptionTypes} SubscriptionTypes
+ */
+
+import { SubscriptionTypes } from '../../../shared/domain/models/SubscriptionTypes.js';
+
+class Subscription {
+  /**
+   * @param {Object} params
+   * @param {number} params.certificationCandidateId - identifier of the subscription
```

Comment:
* @param {number} params.certificationCandidateId - identifier of the subscription
La partie `identifier of the subscription` ne représente pas le paramètre décrit.